### PR TITLE
Migrate CI to github actions

### PR DESCRIPTION
**Pull request by AkshayBhansali18**

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    branches: [ master ]
+
+jobs:
+  Run-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Tests
+        run: bash tests/run_integration_tests.sh
+  openshift-yaml-validate-pgbouncer:
```
Comment (sawood14012) [inline .github/workflows/ci-build.yml:16]: the steps are repeated you can use a matrix startergy here

**Review by sawood14012 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (sawood14012) [reply .github/workflows/ci-build.yml:16]: *Strategy

Comment (AkshayBhansali18) [reply .github/workflows/ci-build.yml:16]: Thank you @sawood14012. will make the required changes

**Review by sawood14012 — APPROVED** [review]
LGTM

**Pull request merged**